### PR TITLE
weather.json addition

### PR DESCRIPTION
**Pull request by dhilipkmr**

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = {
 				items = data[resource];
 				return returnSingle(items, itemArg, res);
 			}
-			if (resource === 'cities') {
+			if (resource === 'cities' || resource === 'weather') {
```
Comment (dhilipkmr) [inline routes/index.js]: Need to return matching cities based in Prefix as well

Comment (kamaljoshi) [reply routes/index.js]: Can you give scenarios where are you going to use it?

Comment (dhilipkmr) [reply routes/index.js]: For example, for city autosuggests where the user types in the city name like "dallas" and he should be getting the availble city names in suggestion when he starts typing.

Comment (dhilipkmr) [reply routes/index.js]: Once a specific city is selected, its weather info will be displayed accordingly

**Pull request merged**